db/seeds.rb

```diff
@@ -4,158 +4,9 @@
-  %w(Katello::KTEnvironment view_lifecycle_environments),
-  %w(Katello::KTEnvironment create_lifecycle_environments),
-  %w(Katello::KTEnvironment edit_lifecycle_environments),
-  %w(Katello::KTEnvironment destroy_lifecycle_environments),
-  %w(Katello::KTEnvironment promote_or_remove_content_views_to_environments),
-  %w(Katello::Product view_products),
-  %w(Katello::Product create_products),
-  %w(Katello::Product edit_products),
-  %w(Katello::Product destroy_products),
-  %w(Katello::Product sync_products),
-  %w(Katello::SyncPlan view_sync_plans),
-  %w(Katello::SyncPlan create_sync_plans),
-  %w(Katello::SyncPlan edit_sync_plans),
-  %w(Katello::SyncPlan destroy_sync_plans),
-  %w(Organization view_subscriptions),
-  %w(Organization attach_subscriptions),
-  %w(Organization unattach_subscriptions),
-  %w(Organization import_manifest),
-  %w(Organization delete_manifest)
-]
-
-permissions.each do |resource, permission|
-  Permission.find_or_create_by_resource_type_and_name resource, permission
-end
-
-default_permissions = {
-  :Viewer => [:view_activation_keys, :view_content_hosts, :view_content_views, :view_gpg_keys, :view_host_collections,
-              :view_lifecycle_environments, :view_products, :view_subscriptions, :view_sync_plans]
-}
-
-Role.without_auditing do
-  default_permissions.each do |role_name, permission_names|
-    permissions = Permission.find_all_by_name permission_names
-    create_filters(Role.find_by_name(role_name), permissions)
-  end
-end
-
-Setting.find_by_name("dynflow_enable_console").update_attributes!(:value => true) if Rails.env.development?
-
-["Pulp", "Pulp Node"].each do |input|
-  f = Feature.find_or_create_by_name(input)
-  fail "Unable to create proxy feature: #{format_errors f}" if f.nil? || f.errors.any?
-end
-
-# Mail Notifications
-notifications = [
-  {:name              => :katello_host_advisory,
-   :description       => N_('A summary of available and applicable errata for your hosts'),
-   :mailer            => 'Katello::ErrataMailer',
-   :method            => 'host_errata',
-   :subscription_type => 'report'
-  },
-
-  {:name              => :katello_sync_errata,
-   :description       => N_('A summary of new errata after a repository is synchronized'),
-   :mailer            => 'Katello::ErrataMailer',
-   :method            => 'sync_errata',
-   :subscription_type => 'alert'
-  },
-
-  {:name              => :katello_promote_errata,
-   :description       => N_('A post-promotion summary of hosts with installable errata'),
-   :mailer            => 'Katello::ErrataMailer',
-   :method            => 'promote_errata',
-   :subscription_type => 'alert'
-  }
-]
-
-notifications.each do |notification|
-  ::MailNotification.find_or_create_by_name(notification)
+katello_seeds = Dir.glob("#{Katello::Engine.root}/db/seeds.d/*.rb"
+katello_seeds.each do |seed|
+  puts "Seeding #{seed}" unless Rails.env.test?
+  load seed
 end
+puts "All katello seed files executed" unless Rails.env.test?
```

Comment:
This file shouldn't be needed anymore -- foreman/config/seeds.rb should handle loading all of these